src/index.js

```diff
@@ -1,12 +1,18 @@
 import 'bootstrap/dist/css/bootstrap.css'
+import { isNullOrUndefined } from 'util';
```

Comment:
What is this? :)

Reply:
It's a js native function to check if an object is null or undefined. Am using it in writeToPTagPrPerson().